build.xml

```diff
@@ -846,6 +851,23 @@
 		<antcall target="run"/>
 	</target>
 
+	<!-- If rewrite-frankframework not installed, run installRecipes.bat -->
+	<target name="install-rewrite" unless="rewrite.installed">
+		<echo message="Installing rewrite-frankframework..."/>
+		<exec executable="${rewrite.dir}/installRecipes.bat" dir="${rewrite.dir}" vmlauncher="false" failonerror="true"/>
```

Comment:
In case a user has the rewrite project cloned today and after two months does a git pull on the rewrite project (which has new changes) should the installRecipes.bat then be called again?